app/views/teachers/classrooms/quizzes/manage_students.html.slim

```diff
@@ -0,0 +1,30 @@
+- content_for :classroom_show_content do
+  h1 = @quiz.name
+  .spacer-small
+
+  .row
+    .large-10.large-push-1.columns
+      table#students-table.display
+        thead
+          tr
+            th Email
+            th Quiz Completed
+            th Actions
+        tbody
+          - @classroom.students.each do |student|
+            tr id="#{dom_id student}"
+              td = student.email
+              td
+                - if student.send(@quiz.name.split.first.downcase + '_quiz_completed') < 0
```

Comment:
The view shouldn't be responsible for knowing that a quiz score of < 0 is not taken. Following the [Single Responsibility Principle](http://www.oodesign.com/single-responsibility-principle.html), the view should only be concerned with what to show to the user.

Knowing that a quiz score of < 0 is considered a quiz not taken is something that belongs to the `Student` model, since that model deals with database-related things.

How about you make a method on the model called `def quiz_taken?(col_name)` and make it return `true` if the score >= 0? Then you can simplify this line to `student.quiz_taken? @quiz.student_column_name` (using the decorator method I describe a couple lines down).